src/Illuminate/Foundation/Console/AppNameCommandException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Exception;
+
+class AppNameCommandException extends Exception
+{
+    public function __construct($message)
```

Comment:
i just have a question here .. why not ? a lot of laravel's exceptions has it's own exception class, i'm trying to understand why you are doing that but your previous comment make me confuse when to make it and when not ?